client/src/components/Input/TextInput.jsx/TextInput.jsx

```diff
@@ -0,0 +1,12 @@
+import './styles.scss';
+
+const TextInput = ({ label, type, placeholder, value, setValue }) => {
+  return (
+    <div className='text-input'>
+      <label>{label}</label>
+      <input type={type} placeholder={placeholder} value={value} onChange={(e) => setValue(e.target.value)}></input>
```

Comment:
Này chỉ là giới thiệu thêm thôi, anh k cần phải sửa nếu k cần thiết
Ngoài những thứ props mặc định như này, còn 1 số thuộc tính khác như autocomplete, disabled,... Những thuộc tính này tùy thuộc vào từng hoàn cảnh, không phải lúc nào cũng sẽ có.
Do đó có thể bổ sung như này
```suggestion
const TextInput = ({ label, type, placeholder, value, setValue, ...rest }) => {
  return (
    <div className='text-input'>
      <label>{label}</label>
      <input type={type} placeholder={placeholder} value={value} onChange={(e) => setValue(e.target.value)} {...rest}></input>
```
Lúc này hoàn toàn có thể bổ sung các thuộc tính đó
`<TextInput autocomplete='off' />`

Reply:
A hiểu rồi. Cám ơn e nhiều nha.